docs/source/api/v3/deliveryservices.rst

```diff
@@ -44,6 +44,8 @@ Request Structure
 	+--------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
 	| tenant       | no       | Show only the :term:`Delivery Services` belonging to the :term:`Tenant` identified by this integral, unique identifier                  |
 	+--------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| topology     | no       | Show only the :term:`Delivery Services` assigned to the :term:`Topology` identified by this unique identifier                           |
```

Comment:
IMO, "identified by this unique identifier" is ambiguous because I don't know if it's a string or a number or what additional meaning it has. More specifically, it is the topology's name. Calling it a unique name might provide clarity.

Reply:
fixed in 277c97e1d5d2490d4d07425dd3f6c3f5559bf0e6